Dockerfile

```diff
@@ -5,7 +5,7 @@
 # Use e.g. --build-arg=base=quay.io/fedora/fedora-bootc:42 to target
 # Fedora instead.
 
-ARG base=quay.io/centos-bootc/centos-bootc:stream10
+ARG base=quay.io/fedora/fedora-bootc:43
```

Comment:
This is a pretty big change of default. I've been trying to ensure that what we do always works on C10S as it's a primary target.

Of course for testing out this PR pre merge that's fine!

Though I think in practice it doesn't really matter because CI is going to do the full matrix anyways...